src/builder/QuarkBuilder.sol

```diff
@@ -178,7 +186,11 @@ contract QuarkBuilder {
             }
 
             if (amountLeftToBridge > 0) {
-                revert FundsUnavailable();
+                revert FundsUnavailable(
+                    transferIntent.amount - balanceOnDstChain,
+                    transferIntent.amount - amountLeftToBridge,
```

Comment:
This should be `requiredAmount - amountLeftToBridge => transferIntent.amount - balanceOnDstChain - amountLeftToBridge` right?

Reply:
oops yeah 😅   forgot the DstChain that don't need to brdige ... oops.